src/JBTokens.sol

```diff
@@ -131,8 +141,15 @@ contract JBTokens is JBControlled, IJBTokens {
         if (tokenOf[projectId] != IJBToken(address(0))) revert PROJECT_ALREADY_HAS_TOKEN();
 
         token = salt == bytes32(0)
-            ? new JBERC20(name, symbol, address(this))
-            : new JBERC20{salt: salt}(name, symbol, address(this));
+            ? IJBToken(Clones.clone(address(TOKEN)))
+            : IJBToken(Clones.cloneDeterministic(address(TOKEN), salt));
```

Comment:
Shouldn't we add an
`salt = keccak256(abi.encode(msg.sender, salt));`

Otherwise you can steal another projects salt by configuring a controller that lets you pass any salt.

Reply:
fixed